validation/src/lib/output.ts

```diff
@@ -42,7 +43,10 @@ export function writeTestcase(comparisonResult: ComparisonResult): CasePreview {
 export function writeSnippetCase(casePreview: CasePreview): void {
   const preview = getPreviewJson();
   preview.snippets.push(casePreview);
-  fs.writeFileSync('./output/preview.json', JSON.stringify(preview));
+  fs.writeFileSync(
+    path.join(__dirname, 'output/preview.json'),
```

Comment:
IIUC this will go to `validation/src/lib/output.preview.json`? I don't think it belongs in `src`, it's data not source code. What was the reason for the move?

Reply:
This move was just for file path convenience, I'll move `output/` to the root directory now as having read your comment I agree it shouldn't be in `src/`.